apps/tester-app/package.json

```diff
@@ -16,16 +16,21 @@
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.23.1",
-    "react": "18.2.0",
-    "react-native": "0.74.3",
-    "react-native-svg": "15.3.0"
+    "react": "18.3.1",
+    "react-native": "0.76.0-rc.1",
+    "react-native-svg": "15.8.0-rc.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.23.9",
+    "@babel/core": "^7.25.2",
+    "@babel/plugin-syntax-typescript": "^7.25.4",
     "@callstack/repack": "workspace:*",
-    "@react-native/babel-preset": "0.74.83",
-    "@react-native/eslint-config": "0.74.83",
-    "@react-native/typescript-config": "0.74.83",
+    "@react-native-community/cli": "15.0.0-alpha.2",
+    "@react-native-community/cli-platform-android": "15.0.0-alpha.2",
+    "@react-native-community/cli-platform-ios": "15.0.0-alpha.2",
+    "@react-native/babel-plugin-codegen": "^0.75.3",
```

Comment:
isn't this included in the babel-preset?

Reply:
No, same thing as above - without it, it crashes. I agree that it should be unneeded since the RN babel preset [already has it](https://github.com/facebook/react-native/blob/v0.76.0-rc.1/packages/react-native-babel-preset/package.json), however still it is not found without this explicit dependency:
<img width="618" alt="image" src="https://github.com/user-attachments/assets/558245f4-9eb9-468d-a148-278ccaf7b92e">